File: kotlin/1672_Richest_Customer_Wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int noOfAccounts = accounts.length;
        int maxWealth = 0;
        for (int i = 0; i < noOfAccounts; i++) {
            int totalWealth = 0;
            int moneyInAccount = accounts[i].length;
            for (int j = 0; j < moneyInAccount; j++) {
                totalWealth = totalWealth + accounts[i][j];
            }
            if (totalWealth > maxWealth) {
                maxWealth = totalWealth;
            }
        }
        return maxWealth;
    }
}

File: java/1672_Richest_Customer_Wealth.kt
class Solution {
    fun maximumWealth(accounts: Array<IntArray>): Int {
        var noOfAccounts = accounts.size
        var maxWealth = 0
        for (account in accounts) {
            var totalWealth = 0
            for (wealth in account) {
                totalWealth = totalWealth + wealth
            }
            if (totalWealth > maxWealth) {
                maxWealth = totalWealth
            }
        }
        return maxWealth
    }
}